<ui version="4.0" >
 <class>SVGProClass</class>
 <widget class="QDialog" name="SVGProClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SVGPro</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3" >
   <property name="geometry" >
    <rect>
     <x>280</x>
     <y>50</y>
     <width>120</width>
     <height>61</height>
    </rect>
   </property>
   <widget class="QRadioButton" name="pixel_choose" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>83</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Pixel</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="cm_choose" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>83</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Cm</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="TitleLabel" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>340</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>30</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title" >
    <string>Image Size</string>
   </property>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>20</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>Height</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>Width</string>
    </property>
   </widget>
   <widget class="QLabel" name="HeightLabel" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>110</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ImageHeight" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>40</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ImageWidth" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="numPlane" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>70</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>70</y>
      <width>82</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Number of Item:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Width of Items:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Height of Items:</string>
    </property>
   </widget>
   <widget class="QLabel" name="WidthLabel" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>90</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>170</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title" >
    <string>View Box Resolution</string>
   </property>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>Width</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>20</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>Height</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="viewBox_Width" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="viewBox_Height" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>40</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="HeightLabelViewBox" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>110</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Height of Items:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Width of Items:</string>
    </property>
   </widget>
   <widget class="QLabel" name="WidthLabelViewBox" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>80</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="view_text" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>180</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>view text</string>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
